miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/InstrumentRestController.java

```diff
@@ -23,66 +23,66 @@
 import org.springframework.web.util.UriComponents;
 import org.springframework.web.util.UriComponentsBuilder;
 
-import uk.ac.bbsrc.tgac.miso.core.data.SequencerReference;
+import uk.ac.bbsrc.tgac.miso.core.data.Instrument;
 import uk.ac.bbsrc.tgac.miso.core.util.PaginatedDataSource;
 import uk.ac.bbsrc.tgac.miso.dto.DataTablesResponseDto;
 import uk.ac.bbsrc.tgac.miso.dto.Dtos;
-import uk.ac.bbsrc.tgac.miso.dto.SequencerDto;
-import uk.ac.bbsrc.tgac.miso.service.SequencerReferenceService;
+import uk.ac.bbsrc.tgac.miso.dto.InstrumentDto;
+import uk.ac.bbsrc.tgac.miso.service.InstrumentService;
 
 @Controller
-@RequestMapping("/rest/sequencer")
-public class SequencerRestController extends RestController {
-  private final JQueryDataTableBackend<SequencerReference, SequencerDto> jQueryBackend = new JQueryDataTableBackend<SequencerReference, SequencerDto>() {
+@RequestMapping("/rest/instrument")
+public class InstrumentRestController extends RestController {
+  private final JQueryDataTableBackend<Instrument, InstrumentDto> jQueryBackend = new JQueryDataTableBackend<Instrument, InstrumentDto>() {
     @Override
-    protected SequencerDto asDto(SequencerReference model) {
+    protected InstrumentDto asDto(Instrument model) {
       return Dtos.asDto(model);
     }
 
     @Override
-    protected PaginatedDataSource<SequencerReference> getSource() throws IOException {
-      return sequencerService;
+    protected PaginatedDataSource<Instrument> getSource() throws IOException {
+      return instrumentService;
     }
   };
 
   @Autowired
-  private SequencerReferenceService sequencerService;
+  private InstrumentService instrumentService;
 
-  public void setLibraryService(SequencerReferenceService sequencerService) {
-    this.sequencerService = sequencerService;
+  public void setLibraryService(InstrumentService instrumentService) {
+    this.instrumentService = instrumentService;
   }
 
-  @RequestMapping(value = "/{sequencerId}", method = RequestMethod.GET, produces = "application/json")
+  @RequestMapping(value = "/{instrumentId}", method = RequestMethod.GET, produces = "application/json")
   @ResponseBody
-  public SequencerDto getById(@PathVariable Long sequencerId) throws IOException {
-    SequencerReference r = sequencerService.get(sequencerId);
+  public InstrumentDto getById(@PathVariable Long instrumentId) throws IOException {
+    Instrument r = instrumentService.get(instrumentId);
     if (r == null) {
-      throw new RestException("No sequencer found with ID: " + sequencerId, Status.NOT_FOUND);
+      throw new RestException("No instrument found with ID: " + instrumentId, Status.NOT_FOUND);
     }
-    SequencerDto dto = Dtos.asDto(r);
+    InstrumentDto dto = Dtos.asDto(r);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "dto". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)